releases/release-6.6.0.md

```diff
@@ -0,0 +1,614 @@
+  </tr>
+  <tr>
+    <td>TiFlash supports stale read</td>
+    <td>TiFlash supports the Stale Read feature, which can improve query performance in scenarios where real-time requirements are not restricted.</td>
+  </tr>
+  <tr>
+    <td rowspan="2">Reliability and Availability<br /></td>
+    <td>Resource control (experimental)</td>
+    <td>Support resource management based on resource groups, mapping database users to the corresponding resource groups and setting quotas for each resource group based on actual needs.</td>
+  </tr>
+  <tr>
+    <td>Historical SQL binding</td>
+    <td>Support binding historical execution plans and quickly binding execution plans on TiDB Dashboard.</td>
+  </tr>
+  <tr>
+    <td rowspan="2">SQL Functionality<br /></td>
+    <td>Foreign key</td>
+    <td>Support MySQL-compatible foreign key constraints to maintain data consistency and improve data quality.</td>
+  </tr>
+  <tr>
+    <td>Multi-valued index (experimental)</td>
+    <td>Introduce MySQL-compatible multi-valued indexes and enhance the JSON type to improve TiDB's compatibility with MySQL 8.0.</td>
+  </tr>
+  <tr>
+    <td>DB Operations and Observability<br /></td>
+    <td>DM support physical import (experimental)</td>
+    <td>TiDB Data Migration (DM) integrates TiDB Lightning's physical import mode to improve the performance of full data migration, with performance being up to 10 times faster.</td>
+  </tr>
+</tbody>
+</table>
+
+## Feature Details
+
+### Scalability
+
+* Support the next generation Partitioned-Raft-KV storage engine (experimental) [#11515](https://github.com/tikv/tikv/issues/11515) [#12842](https://github.com/tikv/tikv/issues/12842) @[busyjay](https://github.com/busyjay) @[tonyxuqqi](https://github.com/tonyxuqqi) @[tabokie](https://github.com/tabokie) @[bufferflies](https://github.com/bufferflies) @[5kbpers](https://github.com/5kbpers) @[SpadeA-Tang](https://github.com/SpadeA-Tang) @[nolouch](https://github.com/nolouch)
+
+    Before TiDB v6.6.0, TiKV's Raft-based storage engine used a single RocksDB instance to store the data of all Regions of the TiKV instance. To support larger clusters more smoothly, starting from TiDB v6.6.0, a new TiKV storage engine is introduced, which uses multiple RocksDB instances to store TiKV Region data, and the data of each Region is independently stored in a separate RocksDB instance. The new engine can better control the number and level of files in the RocksDB instance, achieve physical isolation of data operations between Regions, and support smoothly managing more data. You can see it as TiKV managing multiple RocksDB instances through partitioning, which is why the feature is named as Partitioned-Raft-KV. The main advantage of this feature is better write performance, faster scaling, and larger volume of data supported with the same hardware. It can also support larger cluster scale.
```

Comment:
```suggestion
    Before TiDB v6.6.0, TiKV's Raft-based storage engine used a single RocksDB instance to store the data of all Regions of the TiKV instance. To support larger clusters more stably, starting from TiDB v6.6.0, a new TiKV storage engine is introduced, which uses multiple RocksDB instances to store TiKV Region data, and the data of each Region is independently stored in a separate RocksDB instance. The new engine can better control the number and level of files in the RocksDB instance, achieve physical isolation of data operations between Regions, and support stably managing more data. You can see it as TiKV managing multiple RocksDB instances through partitioning, which is why the feature is named Partitioned-Raft-KV. The main advantage of this feature is better write performance, faster scaling, and larger volume of data supported with the same hardware. It can also support larger cluster scales.
```